### PR TITLE
scatterPlot functionality added

### PR DESCRIPTION
**Pull request by rsanchezgarc**
Scatter plot added to plotter

### CODE REVIEW

File: pyworkflow/em/viewers/plotter.py [hunk 1]
```diff
@@ -78,7 +78,11 @@ def plotAngularDistributionFromMd(self, mdFile, title, **kwargs):
     def plotHist(self, yValues, nbins, color='blue', **kwargs):
         """ Create an histogram. """
         self.hist(yValues, nbins, facecolor=color, **kwargs)
-    
+
+    def plotScatterPlot(self, xValues, yValues, color='blue', **kwargs):
```
Comment (delarosatrevin) [inline pyworkflow/em/viewers/plotter.py]: Can we just name it 'plotScatter' ? 'plotScatterPlot' seems a bit redundant to me.

File: pyworkflow/gui/plotter.py [hunk 2]
```diff
@@ -167,6 +167,7 @@ def createSubPlot(self, title, xlabel, ylabel, xpos=None, ypos=None,
         self.last_subplot = a
         self.plot = a.plot
         self.hist = a.hist
+        self.scatterP= a.scatter
```
Comment (delarosatrevin) [inline pyworkflow/gui/plotter.py]: Please follow PEP8, it should be spaces around = sign.

File: pyworkflow/em/viewers/plotter.py (hunk 1 above)
Comment (rsanchezgarc) [reply pyworkflow/em/viewers/plotter.py]: sure!

**Comment by dmaluenda** [comment]
@rsanchezgarc  keep in mind that this code update can't be used in the plugin until new Scipion's release. Then, if some code in scipion-em-xmipp needs it, it can't be released until new Scipion's release.

**Comment by delarosatrevin** [comment]
@dmaluenda Now Scipion releases will be mainly the core, so I don't expect them to be so sparse as in the past.

**Comment by dmaluenda** [comment]
Right, that's true. Even though, we must take into account that new Scipion code will not be able until it's released.

Of course it seems so obvious, but it's a bit tricky. Because a user can update a plugin just by a single click in the plugin manager. However, the Scipion update is not as easy as that. Then, user can wait more to update the Scipion's core than a certain plugin. Therefore, we need to properly manage the ScipionCore-PluginVersion incompatibilities. I know that it's a plugin-side concern but, just to point it out.

**Pull request merged**